website/docs/jellyfish/api/oracle.md

```diff
@@ -110,11 +110,11 @@ Returns oracle data.
 
 ```ts title="client.oracle.getOracleData()"
 interface oracle {
-  getOracleData (oracleid: string): Promise<OracleData>
+  getOracleData (oracleId: string): Promise<OracleData>
 }
 
 interface OracleData {
-  oracleid: string
+  oracleId: string
```

Comment:
eh no you can't edit interface coming out of `ain` you can only edit what you control

Reply:
I knew. I have changed in the js file. Forgot to add in this md file.